assets/prototype/domain/shared/entities/tickets/predicates/sorters/index.test.ts

```diff
@@ -0,0 +1,72 @@
+import { compareAsc, parseISO } from 'date-fns';
+
+import sorters from './index';
+import { SortTicketsBy } from '../../../../../eventEditor/data/ticket/types';
+import { nodes as tickets } from '../../../../../eventEditor/data/queries/tickets/test/data';
+
+describe('sorters', () => {
```

Comment:
This is duplicating dates sorting, not sure I am :100: happy

Reply:
we can address that later in another PR if need be, although it's likely more trouble than it's worth to try and abstract these tests so that they can be shared. Unless you are talking about abstracting the actual sorting logic so that it can be shared by both tickets and dates, but again, that can be addressed later.

Reply:
Yes, agree this might be a premature abstraction.
I just wanted to share my concern right away, in case someone else might see this and ask why there is a bit of overlap.